app/routes/story.routes.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (app) => {
+    const Story = require("../controllers/story.controller.js");
+    const { authenticateRoute } = require("../authentication/authentication");
+    var router = require("express").Router();
+  
+    // Create a new Recipe
+    router.post("/stories/", [authenticateRoute], Story.create);
+  
+    // // Retrieve all Stories for user
+    // router.get(
+    //   "/stories/user/:storyId",
+    //   [authenticateRoute],
+    //   Story.findAllForUser
+    // );
+  
+    // Retrieve a single Recipe with id
+    router.get("/stories/:storyId", Story.findOne);
+  
+    // Update a Recipe with id
+    router.put("/userId/:storyId", [authenticateRoute], Story.update);
```

Comment:
I'm not sure the endpoint would need to take in both storyId and userId. I think just a storyId would suffice.
```suggestion
    router.put("/stories/:storyId", [authenticateRoute], Story.update);
```